vendor/knative.dev/pkg/reconciler/filter.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Knative Authors
+Copyright 2020 The Knative Authors
```

Comment:
If i am not wrong you may have to send PR to [pkg](https://github.com/knative/pkg) if you want to move filter.go to pkg

Reply:
yes, already did. Still not merged that is why the hold. See https://github.com/knative/pkg/pull/1093

Reply:
Oh sorry i dint see

then i think you no need to do this changes here as part of PR once that is merged it will be taken care here

Reply:
yes, I just change the Gopkg.toml to point to my branch. So that I have the changes, and make things compile. Once pkg is merged, I'll update the toml and it should be good to go...
Did something with a license though... will figure it out